docs/design/network-discovery/port-FSM.puml

```diff
@@ -45,7 +45,11 @@ state OPERATIONAL {
     UP : enter / emit port-up into report
     UP : enter[discovery-enabled] / issue addWatch
     UP : poll-discovery / emit discovery
+    UP : poll-discovery[!last-discovery-was-successful] / emit disable slow-discovery
```

Comment:
avoid using reverted logic for this, last-discovery-was-unsuccessful better for readability

Reply:
At the heartbeat we discussed that we should enable slow discovery only for ports without links. And since PortFsm does not know anything about links, now NetworkUniIslService will make these decisions.

Reply:
I am talking about the text view, not the whole idea.